File: code/69205.pas
program Bai205;
  var a, b,n, dem: longint;
  function SNT(x: longint) : boolean;
    var i :longint;
    begin
    if x < 2 then exit(false);
    for i := 2 to x div 2 do 
        if x mod i = 0 then exit(false);
        exit(true);
    end;
begin
  readln(a, b);
  dem := 0;
  For n := a to b do if SNT(n) = true  then dem := dem + 1;
      writeln(dem);
end.

File: code/69205.cpp
#include <iostream>
using namespace std;

bool snt(int n){
    int x;
    cin>>n;
    if (n<2)
        return false;
    for (x=2; x<=n-1; x++)
        if (n%x==0)
            return false;
    return true;
}

int main() 
{
    int a,b,dem,x;
    cin>>a>>b;
    dem=0;
    for (x=a; x<=b; x++)
        if (snt(x))
            dem++;
    cout<<dem;
    return 0;
}